### ISSUE TITLE
[Discussion] Allowing content above the Proposal Creation form

### ISSUE THREAD

**Issue by DefiDebauchery** [snapshot-labs/snapshot#2128]
**Is your feature request related to a problem? Please describe.**
Many Spaces suffer from low-quality Proposals, which I believe is due in part to an absence of guidelines or other communication being available in the voting portal.

Snapshot's default personalization fields do not provide any way of communicating guidance to users, and according to @mktcode in the Snapshot Discord, could not be extended via plugins, as there is no render area above the Proposal Create form.

**Describe the solution you'd like**
Preferably, I believe all Spaces would benefit from this field being part of Snapshot's base personalization settings. Whether a full list of proposal guidelines, or simply set expectations for what a successful Proposal tangibly means, I believe many-to-most Spaces would utilize this.

However, at least providing an area before the Proposal Create form to be an update target would allow Plugin creators to build it out.

**Comment by mktcode** [comment]
Adding another slot for plugins to render content configured in space/plugin settings would in fact be rather easy to implement. But maybe it would make more sense to add this as core functionality. A plugin might stay unnoticed by many. A new text field in the settings would be more visible. @bonustrack @samuveth @metrox-eth what do you think?

**Comment by samuveth** [comment]
Yes I think adding this to our settings would be best. Another features that could go will with this or could even be an alternative are templates. Similar to github issue templates.

**Comment by bonustrack** [comment]
@samuveth Agree, we can add a field "guidelines" on the space level.

**Comment by mktcode** [comment]
So now there's guidelines vs templates. Guidelines would be rather simple to implement. A new textarea field in the settings and then prepolulate the textarea on proposal creation page. One little question might be whether we want a markdown preview for this settings field or not. 

The template approach would be a bit more involved. It would require a more complex settings area where you can add, remove, edit templates, and also set a default one. Some code could from the strategies/plugins dialogues could be reused maybe. On the creation page the content field needs to be filled with the default template and it needs some dropdown to select a template. And then this would require some logic to make sure things don't get lost when switching the template or being able to reset. (just thinking loud)

So we're probably roughly talking about 1 day vs 1 week.

**Comment by samuveth** [comment]
Yeah I think they should be separate. I'm thinking of guidelines as an URL field in the space settings and a external link block on the proposal creation page. A bit like the discussion field we just added.

**Comment by DefiDebauchery** [comment]
It's been quite a bit since we have discussed this. 

Many spaces still suffer from being able to provide guidance on what constitutes a "good" (or in many cases, "valid") proposal. Having this information front and center when creating a Proposal would greatly reduce resulting strife if such a submission needs to be removed.

Additionally, I still believe it should be a direct Markdown field rather than a link, since such guidelines are central to the platform itself.

Any updates that can be shared, especially if it's "one hour effort"? cc: @mktcode

**Comment by zzuziak** [comment]
@bonustrack shall we change the `Guidelines` field to a Markdown instead? We can suggest rendering an URL (the [ ] ( ) syntax) if a specific space holds their guildelines somewhere else.

**Comment by samuveth** [comment]
@zzuziak  No, settings are already to big when signing. Before we think about this we need to fix settings on hub!

**Comment by zzuziak** [comment]
Is there an issue related to that on Hub? If so, could you link it here @samuveth :pray:

**Comment by samuveth** [comment]
@zzuziak Yes, it's here https://github.com/snapshot-labs/snapshot-hub/issues/391
